packages/mcdoc/src/binder/index.ts

```diff
@@ -1230,11 +1229,16 @@ function convertLiteralValue(
 function convertLiteralNumberSuffix(
 	node: LiteralNode | undefined,
 	ctx: McdocBinderContext,
-): LiteralNumberSuffix | undefined {
+): (typeof NumericTypeKinds)[number] | undefined {
```

Comment:
Can we use `NumericTypeIntKind` instead of `(typeof NumericTypeIntKinds)[number]`?

Reply:
Yep, that's less cursed for sure